src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -235,6 +237,12 @@
             optional = true)
     private Boolean validateSampleToReaderMap = false;
 
+    @Argument(fullName = MERGE_INPUT_INTERVALS_LONG_NAME,
+            shortName = MERGE_INPUT_INTERVALS_LONG_NAME,
+            doc = "Boolean flag to import all data in between intervals.  Improves performance using large lists of " +
+                "intervals, as in exome sequencing, especially if GVCF data only exists for specified intervals.")
+    private boolean mergeInputIntervals = false;
```

Comment:
This looks good... I guess the other alternative would be to add a new interval merging rule to the interval argument collection and push this behavior into the engine.